deployment/sample-workload/execute_workload_flow.sh

```diff
@@ -33,6 +74,40 @@ function makeHttpsCall(){
     eval "$4=$http_code"  
 }
 
+function validateEnvironmentFile(){
+    envFileName=$DEFAULT_FILE_NAME
+    #Read input variable for environment file. If not exist use default file
+    if [ $# -eq 1 ];then
+        envFileName=$1    
+    fi
+    printf "Using $envFileName environment file\n"
+    checkIfFileExists $envFileName
+    if [ $? == 1 ]; then
+        echo -e "\u274c  Environment file ($envFileName) does not exist, exiting... Please create env file and rerun script"
+        return 1
+    fi
+    export $(grep -v '^#' $envFileName | xargs -d '\n')
+    
+    #check if environment file exists
+    hasError=false
+    if [[ -z "$WORKLOAD_URL" ]];then
+        printf "${CODE_ERROR}%s${CODE_NC}" "WORKLOAD_URL must be present"
+        hasError=true
+    fi    
+    if [[ -z "$KBS_URL" ]];then
+        printf "${CODE_ERROR}%s${CODE_NC}" "KBS_URL must be present"
+        hasError=true
+    fi  
+    if [[ -z $KBS_KEY_ID ]];then
+        printf "${CODE_ERROR}%s${CODE_NC}" "KBS_KEY_ID must be present"
+        hasError=true
+    fi  
+    if [[ $hasError == true ]];then
+        printf "For environment variable refer "'\e]8;;%s\e\\%s\e]8;;\e\\\n' "${HELP_LINK}" "${HELP_LINK}"
```

Comment:
How about `For info on environment variable refer...`?

Reply:
Changed